src/commands/Twitch/twitchsubscription.ts

```diff
@@ -188,18 +186,33 @@ export default class extends SkyraCommand {
 
 		// If the streamer was not defined, reset all entries and purge all entries.
 		if (typeof streamer === 'undefined') {
-			const entries = subscriptions.reduce((accumulator, subscription) => accumulator + subscription[1].length, 0);
-			if (entries === 0) throw message.language.tget('COMMAND_TWITCHSUBSCRIPTION_RESET_EMPTY');
+			const subscription = subscriptions.reduce((accumulator, subscription) => accumulator + subscription[1].length, 0);
+			if (subscription === 0) throw message.language.tget('COMMAND_TWITCHSUBSCRIPTION_RESET_EMPTY');
```

Comment:
```suggestion
			const subscriptions = subscriptions.reduce((accumulator, subscription) => accumulator + subscription[1].length, 0);
			if (subscriptions === 0) throw message.language.tget('COMMAND_TWITCHSUBSCRIPTION_RESET_EMPTY');
```

Don't remove the plural for arrays